src/SFA.DAS.EAS.Portal/Database/Models/Account.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using SFA.DAS.EAS.Portal.Client.Models;
+
+namespace SFA.DAS.EAS.Portal.Database.Models
+{
+    public class Account : Document, IAccountDto
+    {
+        [JsonProperty("accountId")]
+        public long AccountId { get; private set; }
+
+        [JsonProperty("accountLegalEntities")]
+        public IEnumerable<IAccountLegalEntityDto> AccountLegalEntities => _accountLegalEntities;
+
+        [JsonProperty("outboxData")]
+        public IEnumerable<OutboxMessage> OutboxData => _outboxData;
```

Comment:
The Outbox is nothing to do with the Account.  It should be specified by a separate Interface and the implementation moved to the base class. Add as tech debt in 454.